.github/workflows/test.yml

```diff
@@ -173,12 +208,20 @@ jobs:
         run: make check-upgrades
 
       - name: Test distribution
-        run: |
-          make dist
-          make distcheck
+        run: make dist
+
+      - name: Test distribution check
+        run: make distcheck
+
+      - name: Build packages
+        uses: linz/linz-software-repository@v8
+        with:
+          release: ${{ matrix.release }}
+        env:
+          PG_SUPPORTED_VERSIONS: ${{ matrix.pg }}
 
       - name: Test debian packaging
```

Comment:
This name doesn't really fit with the command anymore - maybe something like "Install package" would work?